backend/tet/events/tests/data/linked_events_responses.py

```diff
@@ -551,7 +551,8 @@
             "search_vector_sv": "",
             "replaced_by": null,
             "provider": {
-                "fi": "Päiväkoti"
+                "fi": "Test company",
+                "sv": "654321-5"
```

Comment:
"fi" contains company name and "sv" contains business ID, is this correct?

Reply:
This is indeed a bit misleading. Normally we use Linked Event's `custom_data` field for attributes that the Linked Events data model doesn't support. But the problem is that `custom_data` is not searchable, and we need to have these searchable. This is why we use the `provider` object to store this data.

The code uses constants `PROVIDER_NAME_FIELD` and `PROVIDER_BUSINESS_ID_FIELD` to refer to these field. I'll update the test code to also them.